src/ansys/dpf/composites/server_helpers/_upload_files_to_server.py

```diff
@@ -11,6 +13,31 @@
 )
 
 
+def upload_file_to_unique_folder(path_on_client: _PATH, server: BaseServer) -> str:
+    """Upload file to a unique folder on the server.
```

Comment:
Should we make it explicit that this is still a _temporary_ folder? Maybe even include it in the name?